src/services/TestResultsService.ts

```diff
@@ -426,4 +433,38 @@ export class TestResultsService {
     });
     return payload;
   }
+
+  public isTestTypeAdr(testType: any): boolean {
+    let bool = false;
+    const adrTestTypeIds = ["50", "59", "60"];
+    adrTestTypeIds.forEach((id) => {
```

Comment:
can be reduced to 
line 439 and
` return adrTestTypeIds.includes(testType.testTypeId)`

Reply:
modified